host.go

```diff
@@ -176,6 +176,15 @@ func (h *Host) Connect(term *sshd.Terminal) {
 			term.SetPrompt(GetPrompt(user))
 			user.SetHighlight(user.Name())
 		}
+
+		// Gross hack to override line echo in golang.org/x/crypto/ssh/terminal
```

Comment:
You should feel bad.

Reply:
I doooo

Reply:
@Low-power is this going to break your relay?

Reply:
It's not quite clear, is this "dirty hack" in a code path even without the '/timestamps' feature being requested?

Reply:
@Osndok Correct, we fundamentally change how we render your own messages regardless of timestamps. This makes timestamps trivial (and possibly other things in the future).